src/matchers/toInclude/index.js

```diff
@@ -0,0 +1,31 @@
+import { matcherHint, printExpected, printReceived } from 'jest-matcher-utils';
+
+import predicate from './predicate';
+
+const passMessage = (actual, expected) => () => {
```

Comment:
Just noticed this function isn't returning the message if you remove the curly braces it should fix this snapshot:

```
exports[`.toInclude .not.toInclude fails when a string does have a given substring 1`] = `"<red>No message was specified for this matcher.</>"`;
```

```js
const passMessage = (actual, expected) => () =>
  matcherHint('.not.toInclude') +
    '\n\n' +
    'Expected string to not include:\n' +
    `  ${printExpected(expected)}\n` +
    'Received:\n' +
    `  ${printReceived(actual)}`;
```

Reply:
Yup I'll fix that now